clouddriver-artifacts/src/main/java/com/netflix/spinnaker/clouddriver/artifacts/gcs/GcsArtifactCredentials.java

```diff
@@ -50,25 +52,27 @@
     JsonFactory jsonFactory = JacksonFactory.getDefaultInstance();
     String credentialsPath = account.getJsonPath();
 
-    GoogleCredential credential;
+    GoogleCredentials credentials;
 
     if (!StringUtils.isEmpty(credentialsPath)) {
```

Comment:
I know you didn't touch this line, but it seems like this logic could be flipped and easier to read:

```
if (StringUtils.isEmpty(credentialsPath)) {
  credentials = GoogleCredentials.getApplicationDefault();
} else {
  credentials = GoogleCredentials.fromStream(stream)...
}
```

Reply:
Ok, done.